### ISSUE TITLE
[Feature Request] NAME OF THE FEATURE

### ISSUE THREAD

**Issue by jochen-01** [NeoApplications/Neo-Backup#769]
Currently backup intervals are defined in days, only expressed as distance (e.g. every 7 days).
Please add intervals like "every 2 thursdays", "every 3rd wednesday".

**Comment by hg42** [comment]
I'd suggest a start date and an interval.

You often want to create a certain raster filled with different kinds of schedules. And the day you define it, isn't the day where all should start.

E.g. every second week do a full backup, and the week in between do a backup of important apps.

Some want it based on weekdays (e.g. if they have a strong weekly rhythm, like 5 days work, 2 days private), others do things more continuously and don't want a weekly interval, but still want a shifted raster.

A start date plus interval fits all these strategies and allows arbitrary intervals.

**Comment by jochen-01** [comment]
Yes. But I want the backup to take place, let's say, every Sunday. You're right, that is an interval of 7 days. But if I start configuring that on Wednesday I have to count the days 'til Sunday, set the interval to 4 days, wait for Sunday and change it to 7 days. And I noticed problems maintaining the interval when travelling through several time zones.
If you think it's too complicate to code I have to live with what it's like now. But please think about enhancing neo backup in this aspect.

**Comment by hg42** [comment]
sorry, that I wasn't clear.

I agree that something should be added.

I only wanted to say, that we should not add weekdays, but a more general solution, that also fits other needs.
Adding a start date and an interval is a little bit more flexible.

**Comment by jochen-01** [comment]
Yes, that's what I mean. Something like the usual appointment selection we know from, e.g.,  task managers.

**Comment by hg42** [comment]
I think it will be fairly easy to implement, just a new start date field (allowing any date, starting with now), then use that instead of timePlaced, which is used currently for the raster.
Currently there's a small loop to add intervals, this should then change to a calculation.

The only problem is time... :-)

... to implement it and motivation. It's summer ...

**Comment by jochen-01** [comment]
> The only problem is time... :-)
> 
> ... to implement it and motivation. It's summer ...

Now it's autumn.  What does that mean for your motivation? ;-)
